source/features/release-download-count.tsx

```diff
@@ -31,15 +36,15 @@ async function getAssetsForTag(tags: string[]): Promise<Tag> {
 	`);
 
 	const assets: Tag = {};
-	for (const [tag, release] of Object.entries(repository)) {
-		assets[tag] = (release as AnyObject).releaseAssets.nodes;
+	for (const [tag, release] of Object.entries(repository as ReleaseAssets)) {
```

Comment:
I think this type is still wrong. Repository does not look like

```js
{
 	releaseAssets: {
 		nodes: {...}
 	}
 }
```

`ReleaseAssets` seems to be the type of the `release` constant. Please double check, I already commented on this.